### PR TITLE
Improve build required msg

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
Make the message more clear and mention trigger type.

### CODE REVIEW

File: packit_service/constants.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 DOCS_HOW_TO_CONFIGURE_URL = f"{DOCS_URL}/guide/#3-configuration"
 DOCS_APPROVAL_URL = f"{DOCS_URL}/guide/#2-approval"
 DOCS_VM_IMAGE_BUILD = f"{DOCS_URL}/cli/build/in-image-builder/"
-DOCS_TESTING_FARM = f"{DOCS_URL}/testing-farm"
+DOCS_TESTING_FARM = f"{DOCS_URL}/configuration/upstream/tests"
```
Comment (lbarcziova) [inline packit_service/constants.py:16]: :+1:

**Review by lbarcziova — APPROVED** [review]
thanks!

**Comment by lachmanfrantisek** [comment]
Do you think we should update the docs as well?

I am looking at https://packit.dev/docs/configuration/upstream/tests and I don't think it's really needed. The specific error is not mentioned but the build requirement is mentioned.

**Comment by lbarcziova** [comment]
@lachmanfrantisek to me they look ok

**Comment by lachmanfrantisek** [comment]
One more change to fix rendering:
image

**Comment by lachmanfrantisek** [comment]
recheck

**Comment by lachmanfrantisek** [comment]
Zuul does not work fully today..;( Merging manually..

**Pull request merged**